### PR TITLE
Add message for search result is empty

### PR DESCRIPTION
**Pull request by VNTee**

### CODE REVIEW

**Review by goponenkoev — APPROVED** [review]

File: src/test/java/com/epam/service/TourOfferServiceTest.java [hunk 1]
```diff
@@ -141,5 +147,11 @@ public void updateTourThrowExceptionDiscountZero(){
     public void updateTourThrowExceptionTourDescriptiontNull(){
         when(tourOfferService.updateTour(expectedTourOffer,"test",1,1,null)).thenThrow(IllegalArgumentException.class);
     }
+
+    @Test(expected = NotFoundException.class)
+    public void searchTourThrowNotFoundException(){
+        when(tourOfferDAO.searchTours(expectedHotelsId,LocalDate.now(),LocalDate.now())).thenReturn(tourOfferList);
+        when(tourOfferService.searchTours("test",LocalDate.now(),LocalDate.now())).thenThrow(NotFoundException.class);
+    }
```
Comment (Cisfinitum) [inline src/test/java/com/epam/service/TourOfferServiceTest.java:163]: but u have only when().thenSmth(), were u call searchTour?

File: src/main/java/com/epam/service/TourOfferService.java [hunk 2]
```diff
@@ -94,7 +95,12 @@ else if(tourDescription == null){
         for(Hotel hotel: myList){
             listOfHotelsId.add(hotel.getId());
         }
-            return tourOfferDAO.searchTours(listOfHotelsId, startDate, endDate);
+         List<TourOffer> listOfTours = tourOfferDAO.searchTours(listOfHotelsId, startDate, endDate);
+        if(listOfTours.size() == 0){
+            log.error("No tours available");
```
Comment (ghost) [inline src/main/java/com/epam/service/TourOfferService.java]: I believe it is not error, just a regular case. U can use info/warning log level here

**Review by ghost — APPROVED** [review]

File: src/main/java/com/epam/service/TourOfferService.java (hunk 2 above)
Comment (VNTee) [reply src/main/java/com/epam/service/TourOfferService.java]: Yes, you right.

Comment (VNTee) [reply src/main/java/com/epam/service/TourOfferService.java]: Changed level of log to "warn".

**Review by MaltsevaKsenia — APPROVED** [review]

**Review by Cisfinitum — APPROVED** [review]

**Pull request merged**